scss/_patterns_navigation-reduced.scss

```diff
@@ -0,0 +1,84 @@
+@mixin vf-p-navigation-reduced {
+  .p-navigation--reduced {
+    position: relative;
+    z-index: 99; // display above sticky top navigation, but below modals/overlays
+
+    // LOGO OVERRIDES FOR REDUCED NAVIGATION
+
+    // orange logo tag is hidden in reduced navigation
+    .p-navigation__tagged-logo {
+      .p-navigation__link {
+        padding-left: 0;
+      }
+
+      .p-navigation__logo-tag {
+        display: none;
+      }
+    }
+
+    // reduced nav logo text uses default font size (on small screens)
+    .p-navigation__logo-title {
+      font-size: #{map-get($font-sizes, default)}rem;
+    }
+
+    // REDUCED SIZE OF NAVIGATION ON LARGE SCREENS
+    @media (min-width: $breakpoint-navigation-threshold) {
+      background-color: $colors--theme--background-alt;
+
+      // adjust font size for reduced nav on large screens
+      .p-navigation__link,
+      .p-navigation__logo-title {
+        color: $colors--theme--text-muted;
+        font-size: #{map-get($font-sizes, small)}rem;
+        line-height: map-get($line-heights, x-small);
+      }
+
+      .p-navigation__link {
+        padding-bottom: $spv--small;
+        padding-top: $spv--small;
+      }
+
+      .p-navigation__item--dropdown-toggle .p-navigation__link {
+        &::after {
+          top: $spv--small;
+        }
+      }
+
+      .p-navigation__item--dropdown-toggle.is-active {
+        background-color: $colors--theme--background-default;
+      }
+    }
+
+    // SEARCH IN REDUCED NAVIGATION
+
+    .p-navigation__link--search-toggle {
+      // search label is always hidden in reduced navigation
+      .p-navigation__search-label {
+        display: none;
+      }
+
+      @media (min-width: $breakpoint-navigation-threshold) {
+        &::after {
+          top: $spv--small;
+        }
+      }
+    }
+
+    // hide secondary navigation banner when search is open on mobile
+    &.has-search-open + .has-search-open .p-navigation__banner {
```

Comment:
This is a bit of a weird case, because there is no shared parent element and we need to kind of share the state between two navigations (top reduced one, and secondary one below it).
Basically the top navigation has a search toggle, that opens the search in secondary nagivation. And in such case both have `has-search-open` class name.

It's not different on mobile or desktop, so it's not a matter of having 2 class names.